tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplatesTest.cs

```diff
@@ -110,6 +110,25 @@ public void TestMultiLineExprInLG()
             Assert.IsTrue(evaled == "15", $"Evaled is {evaled}");
         }
 
+        [TestMethod]
+        public void TestExpandText()
+        {
+            var templates = Templates.ParseFile(GetExampleFilePath("ExpandText.lg"));
+
+            var scope = new JObject
+            {
+                ["@answer"] = "hello ${user.name}",
+                ["user"] = new JObject
+                {
+                    ["name"] = "vivian"
+                }
+            };
+
+            // - ${expandText(@answer)}
+            var evaled = templates.Evaluate("template", scope);
+            Assert.AreEqual("hello vivian", evaled);
+        }
```

Comment:
Would be good to have a top-level test of expandText and not just embedded in a template.